controller/deploy_controller.rb

```diff
@@ -15,10 +18,13 @@ def initialize(environ, git_helper, user_comms, develop_tag_generator, other_tag
   def environment_choice
     @environ.downcase
```

Comment:
Usually in Ruby, we use setters and getters internally, too.

So you define attributes with for example
`attr_accessor :environ`

and access it with
`self.environ = environ`
and
`environ.downcase`